project-management/README.md

```diff
@@ -0,0 +1,95 @@
+# Project Management
+
+<img align="right" width="100" height="100" alt="Lingua Pupuli Logo" src="https://avatars3.githubusercontent.com/u/37545262?s=200&v=4">
+
+We use Github [Issues](https://help.github.com/articles/about-issues/), [Milestones](https://help.github.com/articles/about-milestones/) and [Projects](https://help.github.com/articles/about-project-boards/) to track and mange the work for the organization
+
+* [Project](#Projects)
+* [Milestones](#Milestones)
+* [Issues](#Issues)
+* [Pull Requests](#Pull%20Requests)
+
+## Projects
+
+Projects are used to track complicated work in a single repository, or to coordinate work across multiple repositories
+
+Only two types of projects are allowed: **Releases** and **Features**
+
+### Why?
+
+Provides an easy to track long lived features or cross repository task
+
+### Releases
+
+* Correlate to VSCode marketplace releases
+* Encompass all work done for that release
+* Can contain issues/prs from just one, or more than one repo
+
+### Features
+
+* Do not correlate to any release, can span releases
+* Contains issues and PRs that encompass work done in more than one repository
+* Does not contain issues/prs that are only in one repo
+* Can live longer than a Release project
+
+
+## Milestones
+
+Milestones are used to track the work of a single release of a repository
```

Comment:
It would be useful to have an example / clarification of what fits here - version milestones only? milestones named for small features? I _intuit_ it's the former, but just want to be sure.

Reply:
Maybe clarify something like `for example, a release of the vscode-client might have version 0.15.0 as the milestone`

Reply:
Versioned milestones. Features that span repos are projects, features that don't are issues that get put into milestones.